keras_contrib/losses/jaccard.py

```diff
@@ -28,7 +28,13 @@ def jaccard_distance(y_true, y_pred, smooth=100):
     https://en.wikipedia.org/wiki/Jaccard_index
 
     """
-    intersection = K.sum(K.abs(y_true * y_pred), axis=-1)
-    sum_ = K.sum(K.abs(y_true) + K.abs(y_pred), axis=-1)
+    if smooth is None:
+        smooth = K.epsilon()
+    intersection = K.sum(K.abs(y_true * y_pred), axis=axis)
+    sum_ = K.sum(K.abs(y_true) + K.abs(y_pred), axis=axis)
     jac = (intersection + smooth) / (sum_ - intersection + smooth)
     return (1 - jac) * smooth
```

Comment:
I believe that you need to remove multiplication to smooth here.
So we return only ```1-jac```